packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -109,6 +112,9 @@ export const LivenessCameraModule = (
 
   const [state, send] = useLivenessActor();
 
+  const isFaceMovementChallenge =
+    useLivenessSelector(selectChallengeType) === 'FaceMovementChallenge';
```

Comment:
Can we store this as a constant somewhere?

Reply:
You mean store a boolean for whether or not we're using that challenge?

Reply:
Oh sorry i just meant `FaceMovementChallenge`

Reply:
Ok cool, we're already exporting a constant for that so updated to actually use it lol